packages/slidev/node/plugins/transformSnippet.ts

```diff
@@ -0,0 +1,128 @@
+import path from 'node:path'
```

Comment:
Here in this file, can we have a reference to VitePress' implementation source code? Put some credits while make easy to track with the upstream updates.